docs/Documentation/Scripting/Building-Blocks/Events-List.md

```diff
@@ -606,14 +606,28 @@ events:
     
 ## Give experience: `experience`
 
-Gives the specified amount of experience points to the player. You can give levels by adding the `level` argument.
-You can also define decimal numbers when giving levels, for example `experience 1.5 level` will give 1 level and half.
+With this event you can manipulate the experience of the player. First specify the amount, then the modification type.
+You can use `add_experience`, `add_level`, `set_experience_bar` and`set_level` as modification typs.
```

Comment:
Please don't use snake_case, that is not a format that has been used in the instruction so far.
I would suggest using a `key:value` approach here.
`action:addExperience`, `action:addLevel`, `action:setExperienceBar`, `action:setLevel`